src/navsat_transform.cpp

```diff
@@ -45,7 +45,6 @@ namespace RobotLocalization
 {
   NavSatTransform::NavSatTransform() :
     magnetic_declination_(0.0),
-    utm_odom_tf_yaw_(0.0),
```

Comment:
is this just a cleanup?

Reply:
Yes, it's not used, so I removed when I was building with `clang`, which was failing on that.